django_netjsongraph/base/admin.py

```diff
@@ -164,7 +164,7 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
 
 class AbstractLinkAdmin(BaseAdmin):
     raw_id_fields = ['source', 'target']
-    list_display = ['__str__', 'topology', 'status', 'cost', 'cost_text']
+    list_display = ['__str__', 'topology', 'status', 'cost', 'cost_text', 'uptime_or_downtime']
```

Comment:
let's not add this information in the admin list, we need to show this in the visualizer